openstack/victoria/ovs-kernel/nova.patch

```diff
@@ -1,8 +1,34 @@
 diff --git a/nova/network/os_vif_util.py b/nova/network/os_vif_util.py
-index 0bfd87bb1e..ee6578dcbb 100644
+index 0bfd87bb1e..e7c3c57593 100644
 --- a/nova/network/os_vif_util.py
 +++ b/nova/network/os_vif_util.py
-@@ -338,6 +338,21 @@ def _nova_to_osvif_vif_ovs(vif):
+@@ -25,6 +25,7 @@ from oslo_log import log as logging
+ from nova import exception
+ from nova.i18n import _
+ from nova.network import model
++from nova.pci import utils as pci_utils
+ 
+ 
+ LOG = logging.getLogger(__name__)
+@@ -326,8 +327,16 @@ def _nova_to_osvif_vif_bridge(vif):
+ 
+ # VIF_TYPE_OVS = 'ovs'
+ def _nova_to_osvif_vif_ovs(vif):
+-    vif_name = _get_vif_name(vif)
++    vif_name = ""
+     vnic_type = vif.get('vnic_type', model.VNIC_TYPE_NORMAL)
++    if vnic_type == model.VNIC_TYPE_VIRTIO_FORWARDER:
++        # set the vif_name to be the vif net device, as it will be target dev
++        # for the vdpa ports in the xml file
++        vif_name = pci_utils.get_ifname_by_pci_address(
++                vif["profile"]['pci_slot'])
++    else:
```

Comment:
why do we need else part?

Reply:
I don't want to break something else, it was set before
https://github.com/openstack/nova/blob/master/nova/network/os_vif_util.py#L329